packages/pilot/src/components/Operations/operationsTableColumns.js

```diff
@@ -77,24 +81,76 @@ const renderTargetOrSource = (type, net, targetId, sourceId, labels) => {
   return null
 }
 
+const greaterThanZero = gt(__, 0)
+
+const isFailedTransfer = allPass([
+  either(
+    propEq('type', 'credito_em_conta'),
+    propEq('type', 'transfer')
+  ),
+  propEq('status', 'failed'),
+  pipe(
+    prop('amount'),
+    greaterThanZero
+  ),
+])
+
+const getOutgoingAmount = pipe(
+  head,
+  prop('amount')
+)
+
+const renderCreditTransferStatus = (
+  type, status, outgoing, transferId, labels
+) => {
+  const amount = getOutgoingAmount(outgoing)
+
+  const hasFailed = isFailedTransfer({ amount, status, type })
+  if (hasFailed) {
```

Comment:
não deveria ter uma quebra de linha aqui?
```suggestion
   const hasFailed = isFailedTransfer({ amount, status, type }) 

  if (hasFailed) {
```

Reply:
Boa, feito!